### DISCUSSION TITLE
Text Selection in .epub files

### DISCUSSION THREAD

**Discussion by WebMechanic** [sumatrapdfreader/sumatrapdf#2436] [Q&A] [answered]
Hello,
I love this app, but there is one thing that's truly buggin' me (using v3.2 64bit version on Win10)

Maybe I am missing a setting, but while I can select and copy text from any PDF (unless explicitly prohibited), this seems impossible when opening any .epub file whatsoever: event he ones I compiled myself.

Is there is a setting to enable this I'd much appreciate to learn how to enable it, and if not: why is it prohibited to select text within epub but not other formats?
There is no copy protection of the contents, as I can (and out of frustration have) simply unzip the file, open the HTML sources and go from there.

It also seems to be impossible to print an .epub, while printing .pdf works.

Thanks.

**Comment by GitHubRulesOK** [comment, marked as answer]
Older 3.X versions used a reflowing interface by default for eBooks in general, thus for features such as copy or search the fixed UI is needed and you can in 3.2 set that in advanced settings by changing to UseFixedPageUI = true
 for chm and eBooks

In 3.4 (pre-release) https://www.sumatrapdfreader.org/prerelease that option is only kept for a few types and only remaining control is
```
ChmUI [
	UseFixedPageUI = false
]
```
If you wanted a mix of both features you could use a copy of two official portable.exe's in one folder just name one of them differently however that could cause multiple problems as sharing the same settings thus highly NOT recommended 

The better method is to keep portable exe in 2 or more separate folders where each has its own local settings, but then will not share usage history.

**Comment by WebMechanic** [comment]
aah! great!
That's why there wasn't a toolbar either.
Thank you.
